server/src/infra/auth_service.rs

```diff
@@ -390,6 +455,92 @@ where
     HttpResponse::Ok().finish()
 }
 
+async fn conventional_register<Backend>(
```

Comment:
As mentioned above, let's split that into another PR

Reply:
That really sucks, because I actually need to register users via API without the opaque protocol

Reply:
I'm not saying no, I'm saying that we'll talk about it in the next PR :) I can probably easily be convinced.

Reply:
OK, I moved it to a separate branch